docs/references/general/rawacf.md

```diff
@@ -1,3 +1,110 @@
+| *scan*                  | *None*   | short     | Scan flag, switches between 0 to +/-1. For some radars, the sign indicates scan direction. |
+| *offset*                | *us*     | short     | Offset between channels for a stereo radar                                              |
+| *rxrise*                | *us*     | short     | Receiver rise time                                                                      |
+| *intt.sc*               | *s*      | short     | Number of seconds of integration time                                                   |
+| *intt.us*               | *us*     | int       | Number of microseconds of integration time                                              |
+| *txpl*                  | *us*     | short     | Transmitted pulse length                                                                |
+| *mpinc*                 | *us*     | short     | Multi-pulse increment                                                                   |
+| *mppul*                 | *None*   | short     | Number of pulses in the sequence                                                        |
+| *mplgs*                 | *None*   | short     | Number of lags estimated from the sequence                                              |
+| *mplgexs*               | *None*   | short     | Number of lags used for tauscan. This field originates from `fittex` algorithm and is not used in RAWACF file format. | 
+| *ifmode*                | *None*   | short     | A flag indicating that the receiver voltage samples were obtained by digitizing the signal at an intermediate frequency (and then digitally mixed to baseband) instead of mixing to baseband and then digitizing (`ifmode=0`).         |
+| *nrang*                 | *None*   | short     | Number of range gates                                                             |
+| *frang*                 | *km*     | short     | Distance to the first range gate                                                        |
+| *rsep*                  | *km*     | short     | Range seperation                                                                        |
+| *xcf*                   | *None*   | short     | XCF flag, indicates if XCF for elevation angle estimation is available (1) or not (0)   |
+| *tfreq*                 | *kHz*    | short     | Transmitted frequency                                                                   |
+| *mxpwr*                 | *dB*     | int       | Maximum power                                                                           |
+| *lvmax*                 | *None*   | int       | Maximum noise level allowed                                                             |
+| *combf*                 | *None*   | string    | Comment buffer, usually contains the control program name or command line command used to generate the file                            |
```

Comment:
```suggestion
| *combf*                 | *None*   | string    | Comment buffer, usually contains the control program name                            |
```

As far as I am aware, there are no RST routines which modify the comment buffer (except maybe `make_sim`) - I think that only occurs on-site via the control program.  Command line commands used to generate the file are stored in the `origin.command` field.